spring-security-cors/src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+server.port=8080
```

Comment:
8080 is the default port, right? So, let's take this file out.

Reply:
Removed this file